introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Valentin. I'm Python/JS developer and game/puzzle designer.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.